docs/docs/more-info/architecture.mdx

```diff
@@ -0,0 +1,83 @@
+
+You can think of use case packages as adding features to the end user whereas infrastructural packages provide the dependencies to the use case packages. This separation allow an infrastructural package that, say implements a FHIR server to be replaced with a different packages that implement a different FHIR server. As long as these packages can be configured in a standards based way, a use case packages could work with either of these infrastructural packages. This gives user options for the applications that they wish to use.
+
+<div class="text--center">
+  <img alt="Package architecture" src={useBaseUrl('img/instant-openhie-package-arch.png')} />
+</div>
+
+Each package will contain the following sorts of technical artifacts that allow it to be spun up and down within the supported platforms:
+* Docker Compose scripts for setting up the applications required for this package’s use cases and workflows
+* Kubernetes scripts for setting up the applications required for this package’s use cases and workflows
+* Configuration scripts to setup required configuration metadata
+* Extensions to the test harness to test the added use cases with test data
+
+The exact artefacts that go into a package are described in the section on [creating packages](../how-to/creating-packages.mdx).
+
+# Execution architecture
+
+Much of the tooling that Instant OpenHIE provides is to allow packages to be executed once they have been defined. The core principles of the execution architecture are that:
+
+* Packages may be executed in a cross platform way (on Window, OSX or Linux)
+* Complexity should be hidden from the user to make spinning up packages easy
+* The software that executes the packages should be self contained so that it is easy to download and get running
+
+The core concept of the execution architecture is that all the tooling to execute packages in contained within a single docker image - called the execution image. This enables it to run cross platform as long as the required dependencies are installed (Docker Desktop for Windows/OSX or Docker engine and Docker compose for Linux). To enable the execution image to spin up packages using docker it is passed the docker socket file from the host so that any containers it created are in fact created on the host. It may also be passed other configuration form the host for other platform such as Kubernetes to allow it to deploy packages to those platform as well using the users current cluster configuration. The execution image give us a consistent environment to develop for and allow us to run the scripts that execute the packages without caring about the underlying host OS, we just need to interface with the deployment platform.
```

Comment:
```suggestion
The core concept of the execution architecture is that all the tooling to execute packages contained within a single docker image - called the execution image. This enables it to run cross-platform as long as the required dependencies are installed (Docker Desktop for Windows/OSX or Docker engine and Docker compose for Linux). To enable the execution image to spin up packages using docker it is passed the docker socket file from the host so that any containers it created are in fact created on the host. It may also be passed other configuration form the host for other platforms such as Kubernetes to allow it to deploy packages to those platforms as well using the user's current cluster configuration. The execution image gives us a consistent environment to develop for and allow us to run the scripts that execute the packages without caring about the underlying host OS, we just need to interface with the deployment platform.
```